packages/key-did-resolver/babel.config.js

```diff
@@ -0,0 +1,10 @@
+// babel.config.js
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {targets: {node: 'current'}}],
```

Comment:
`node: 12` maybe to avoid potential issues?

Reply:
makes sense 👍